MonoGame.Framework.Content.Pipeline/Serialization/Intermediate/ReflectiveSerializer.cs

```diff
@@ -127,6 +134,11 @@ protected internal override void Initialize(IntermediateSerializer serializer)
             }
         }
 
+        public override bool CanDeserializeIntoExistingObject
+        {
+            get { return TargetType.IsClass; }
```

Comment:
Okay, I think I'm getting myself mixed up now. We should probably just copy XNA's behaviour, even though it's not that helpful. Otherwise we'll find ourselves having to check the actual type returned from every getter-only property when deserializing, to see whether the XML element is required or not.